packages/bsync/src/context.ts

```diff
@@ -42,16 +42,27 @@ export class AppContext {
       poolIdleTimeoutMs: cfg.db.poolIdleTimeoutMs,
     })
 
-    let revenueCatClient: RevenueCatClient | undefined
-    if (cfg.revenueCat) {
-      revenueCatClient = new RevenueCatClient({
-        v1ApiKey: cfg.revenueCat.v1ApiKey,
-        v1ApiUrl: cfg.revenueCat.v1ApiUrl,
-        webhookAuthorization: cfg.revenueCat.webhookAuthorization,
+    let purchasesClient: PurchasesClient | undefined
+    if (cfg.purchases) {
+      purchasesClient = new PurchasesClient({
+        revenueCatV1ApiKey: cfg.purchases.revenueCatV1ApiKey,
+        revenueCatV1ApiUrl: cfg.purchases.revenueCatV1ApiUrl,
+        revenueCatWebhookAuthorization:
+          cfg.purchases.revenueCatWebhookAuthorization,
+        stripePriceIdMonthly: cfg.purchases.stripePriceIdMonthly,
+        stripePriceIdAnnual: cfg.purchases.stripePriceIdAnnual,
+        stripeProductIdMonthly: cfg.purchases.stripeProductIdMonthly,
+        stripeProductIdAnnual: cfg.purchases.stripeProductIdAnnual,
```

Comment:
Is the reason these are configurable and the other platforms are not because these values will change depending on if we're in test mode or production?

Reply:
Well we can still hard-code them because they won't change after we create them, but I see your point. We can deal with more config when the time comes, or move to the hard-coded option then too if we want.